manual/development-process/production-deployment.md

```diff
@@ -0,0 +1,175 @@
+# Deploy code to production
+
+Transfer Digital Records uses the practice of [Continuous Integration]: developers merge their work into the a repo's
+main branch regularly. The repo is configured to only allow changes to be merged if the checks pass, such as unit tests,
+code linters and security checks. These checks are currently run on a [Jenkins instance][tdr-jenkins-intg].
+
+We also perform [Continous Delivery]. The main branch of any repo is always deployable, and can be deployed to
+production multiple times a day if necessary. The [end-to-end tests] which run on integration and staging help us catch
+regression before code is deployed to production. Deploying frequent changes to production means that each deployment is
+small, which makes deployments less risky and easier to fix.
+
+TDR is made up of [multiple services][tdr-architecture], each of which are deployed separately. Infrastructure is also
+defined in Terraform repos, and is deployed to each environment in the same way way.
+
+## When to deploy
+
+You should deploy to Staging and Production as soon as possible after merging the change and checking it on Integration.
+This means that you can deploy your own change. If you leave your change undeployed and someone else deploys it at the
+same time as theirs, and failures it will be harder for them to debug.
+
+We have agreed to some rules about when it is OK to deploy to Production:
+
+- Only when you have time to monitor the deployment. Decide how long you need based on the size and complexity of the
+  change.
+- Only when there are other people around. We are a small team with flexible working hours, so there are no restrictions
+  based on the time of day, but you should make sure that at least one other developer is around to help debug any
+  deployment issues.
+- Not on a Friday or the day before a bank holiday. Ideally, we would also deploy on Fridays to avoid a build-up of
+  changes that would make the deployment larger, but we need to demonstrate that our deployments are low-risk and that
+  we can catch issues early with tests and monitoring.
+
+These rules only apply to Production, but it's usually better to apply them to Staging as well. This means that you can
```

Comment:
Reword: ... means that you can deploy to Production shortly after ...

Reply:
Fixed

Reply:
Just a full stop missing after "If the end-to-end tests passed on Staging after the most recent deployment"